src/services/identity/__tests__/SitesService.spec.ts

```diff
@@ -302,8 +304,8 @@ describe("SitesService", () => {
   describe("getSites", () => {
     it("Filters accessible sites for github user correctly", async () => {
       // Store the API key and set it later so that other tests are not affected
-      const currRepoCount = process.env.ISOMERPAGES_REPO_PAGE_COUNT
-      process.env.ISOMERPAGES_REPO_PAGE_COUNT = "3"
+      const currRepoCount = config.get("sites.pageCount")
```

Comment:
We are getting and immediately setting it, so I thought it can be placed together for clarity as they go together as a "pair". Alternatively, we can define a single `currRepoCount` at the top and then only do the `set` in here.
```
const currRepoCount = config.get("sites.pageCount")
config.set("sites.pageCount", 3)
```

Reply:
@seaerchin Example if you change it to use `import` syntax in `src/services/fileServices/MdPageServices/MediaFileService.js`:

```
import config from "@config/config"

```

you will get the following error when running tests:

```
 FAIL  src/services/fileServices/MdPageServices/__tests__/MediaFileService.spec.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/harishv/Documents/Code/isomer/isomercms-backend/src/services/fileServices/MdPageServices/MediaFileService.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import config from "@config/config"
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      39 |   const {
      40 |     MediaFileService,
    > 41 |   } = require("@services/fileServices/MdPageServices/MediaFileService")
         |       ^
      42 |   const service = new MediaFileService({
      43 |     gitHubService: mockGithubService,
      44 |   })

      at Runtime.createScriptFromCode (../node_modules/jest-runtime/build/index.js:1728:14)
      at services/fileServices/MdPageServices/__tests__/MediaFileService.spec.js:41:7
      at Object.<anonymous> (services/fileServices/MdPageServices/__tests__/MediaFileService.spec.js:7:1)
```